core/controllers/topics_and_skills_dashboard.py

```diff
@@ -147,6 +147,11 @@ class NewSkillHandler(base.BaseHandler):
     def post(self):
         description = self.payload.get('description')
         linked_topic_ids = self.payload.get('linked_topic_ids')
+        rubrics = self.payload.get('rubrics')
+        if not isinstance(rubrics, list):
+            raise self.InvalidInputException
```

Comment:
Add argument with an error message (see other InvalidInputExceptions for examples).

Reply:
Done